javascript/node/error-handling/README.md

```diff
@@ -2,10 +2,12 @@ name: Error Handling
 
 type: insights-list
 
-description: How to handle various errors. 
+description: How to handle various errors.
 
 section: 1
 
+parent: operational-vs-programmer-errors
+
 insights:
```

Comment:
Let's actually just merge these workouts, call it error handling. Use this ordering:
- operational-vs-programmer-errors
- handling-operational-errors
- handling-programmer-errors
- try-catch-only-for-sync-code
- first-error-callbacks-in-node